data/coaches.yml

```diff
@@ -0,0 +1,125 @@
+-
+  name: Silvana Precup
+  description: Secretary
+  image: silvana.jpg
```

Comment:
Do we still need all the `description` and `image` keys, if we don't use them in the page?